packages/frameworks/test/frameworks.unit.test.ts

```diff
@@ -1,7 +1,8 @@
 import Ajv from 'ajv';
-import path from 'path';
+import { join } from 'path';
 import { existsSync } from 'fs';
 import { Framework } from '../';
+const frameworkList = require('../frameworks.json') as Framework[];
 
 function isString(arg: any): arg is string {
```

Comment:
Not really related to this PR, but could use `util.isString()` here instead.

Reply:
https://github.com/vercel/vercel/pull/4836